migrations/Version20240217103834.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240217103834 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
Add description of what this will bring?

Reply:
Will do